qa/qa-captain/bashrc

```diff
@@ -0,0 +1,1116 @@
+    sudo cp nats-server-v2.8.2-linux-amd64/nats-server /usr/bin/
+    nats-server -v
+}
+
+# --- install krew plugin for kubectl ---
+function get.krew() {
+    # check for git installed
+    (
+  set -x; cd "$(mktemp -d)" &&
+  OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+  ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
+  KREW="krew-${OS}_${ARCH}" &&
+  curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/${KREW}.tar.gz" &&
+  tar zxvf "${KREW}.tar.gz" &&
+  ./"${KREW}" install krew
+)
+}
+
+# --- install krew plugin for kubectl ---
+function get.kuttl() {
+    kubectl krew install kuttl
+}
+# ======================================================================================
+# ================================== ~ CAPTAINS ~ ======================================
+# ======================================================================================
+
+# --- check CI for releases ---
+function get.ci() {
+    #requires drone logins :/ 
+    printf "https://drone-publish.rancher.io/rancher/rke2  or https://drone-publish.k3s.io/k3s-io/k3s"
+}
+
+# --- check DockerHub system agent installers and upgrade images ---
+function get.artifacts() {
+    #requires jq
+    local PRODUCT="${1:-k3s}"
+    local BRANCH="${2:-v1.22}"
+        printf "===== System Agent Installers ===== \n"
+        curl -L -s "https://registry.hub.docker.com/v2/repositories/rancher/system-agent-installer-${PRODUCT}/tags?page_size=300" | jq -r ".results[].name" | sort --version-sort | grep -i -e "$BRANCH" | tail -n 5 # > system-agent-installers.txt
+        printf "===== Upgrade Images ===== \n"
+        curl -L -s "https://registry.hub.docker.com/v2/repositories/rancher/${PRODUCT}-upgrade/tags?page_size=300" | jq -r ".results[].name" | sort --version-sort | grep -i -e "$BRANCH" | tail -n 5 #  > upgrade-images.txt
+        printf '===== %s Images ===== \n' "$PRODUCT"
+    case "$PRODUCT" in
+    rke2)
+        curl -s -H "Accept: application/vnd.github+json" https://api.github.com/repos/rancher/rke2/releases | jq '.[].tag_name' | sort --version-sort | grep -i -e "$BRANCH" | tail -n 1
+        # diff --color -s --suppress-common-lines system-agent-installers.txt upgrade-images.txt
+    ;;
+    k3s)
+        curl -s -H "Accept: application/vnd.github+json" https://api.github.com/repos/k3s-io/k3s/releases | jq '.[].tag_name' | sort --version-sort | grep -i -e "$BRANCH" | tail -n 1
+        # diff --color -s --suppress-common-lines system-agent-installers.txt upgrade-images.txt
+    ;;
+    esac
+}
+
+# --- check for upgrade images in 
+
+
+# ======================================================================================
+# ================================== ~ GITHUB ~ =======================================
+# ======================================================================================
+
+# --- list issues in the github repos using the gh cli --- 
+ function ghls() {
```

Comment:
Formatting seems to be messed up from here down.

Reply:
You're correct - The number of functions within sections drop off and it's a bit of a junk drawer for functions to share with the team to add to their workstation shell environments. The sections CAPTAINS and GITHUB are both for that purpose.